common/core-declarative/src/main/kotlin/br/com/zup/beagle/widget/core/FlexBuilder.kt

```diff
@@ -0,0 +1,197 @@
+
+    fun direction(type: Direction) = this.apply { this.direction = type }
+
+    fun display(type: FlexDisplay) = this.apply { this.display = type }
+
+    fun flex(value: Double) = this.apply { this.flex = value }
+
+    fun flexDirection(type: FlexDirection) = this.apply { this.flexDirection = type }
+
+    fun flexWrap(type: FlexWrap) = this.apply { this.flexWrap = type }
+
+    fun grow(value: Double) = this.apply { this.grow = value }
+
+    fun justifyContent(type: JustifyContent) = this.apply { this.justifyContent = type }
+
+    fun shrink(value: Double) = this.apply { this.shrink = value }
+
+    fun positionType(type: FlexPositionType? = null) = this.apply { this.positionType = type }
+
+    fun basis(value: UnitValue) = this.apply { this.basis = value }
+
+    fun size(size: Size) = this.apply { this.size = size }
+
+    fun position(position: EdgeValue) = this.apply { this.position = position }
+
+    fun padding(padding: EdgeValue) = this.apply { this.padding = padding }
+
+    fun margin(margin: EdgeValue) = this.apply { this.margin = margin }
+
+    fun size(
+        width: UnitValue? = null,
+        height: UnitValue? = null,
+        aspectRatio: Double? = null,
+        maxWidth: UnitValue? = null,
+        minWidth: UnitValue? = null,
+        maxHeight: UnitValue? = null,
+        minHeight: UnitValue? = null
+    ) = this.size(
+        Size(
+            width = width,
+            maxWidth = maxWidth,
+            minWidth = minWidth,
+            height = height,
+            maxHeight = maxHeight,
+            minHeight = minHeight,
+            aspectRatio = aspectRatio
+        )
+    )
+
+    fun position(
+        all: UnitValue? = null,
+        top: UnitValue? = null,
+        bottom: UnitValue? = null,
+        right: UnitValue? = null,
+        left: UnitValue? = null,
+        start: UnitValue? = null,
+        end: UnitValue? = null,
+        horizontal: UnitValue? = null,
+        vertical: UnitValue? = null
+    ) = this.apply {
+        this.position = EdgeValue(
+            all = all,
+            top = top,
+            right = right,
+            bottom = bottom,
+            left = left,
+            start = start,
+            end = end,
+            horizontal = horizontal,
+            vertical = vertical
+        )
+    }
+
+    fun padding(
+        all: UnitValue? = null,
+        top: UnitValue? = null,
+        right: UnitValue? = null,
+        bottom: UnitValue? = null,
+        left: UnitValue? = null,
+        start: UnitValue? = null,
+        end: UnitValue? = null,
+        horizontal: UnitValue? = null,
+        vertical: UnitValue? = null
+    ) = this.apply {
+        this.padding = EdgeValue(
+            all = all,
+            vertical = vertical,
+            horizontal = horizontal,
+            top = top,
+            right = right,
+            bottom = bottom,
+            left = left,
+            start = start,
+            end = end
+        )
+    }
```

Comment:
Why not call the overloaded version that receives `EdgeValue`?
```suggestion
    ) = this.padding(
        EdgeValue(
            all = all,
            vertical = vertical,
            horizontal = horizontal,
            top = top,
            right = right,
            bottom = bottom,
            left = left,
            start = start,
            end = end
        )
    )
```